python/settings_client.py

```diff
@@ -13,6 +13,8 @@
 
 from __future__ import absolute_import, print_function
 
+from six.moves import input
```

Comment:
should probably add 'client' requirements somewhere. tried running this in a fresh `venv` and had to install `sbp`, `six` and `piksi_tools`. Not a big deal but would be helpful

Reply:
Thanks for testing, I'll keep that in mind.